modules/Emails/EmailsController.php

```diff
@@ -139,19 +139,51 @@ public function action_ComposeView()
      */
     public function action_send()
     {
-        $this->bean = $this->bean->populateBeanFromRequest($this->bean, $_REQUEST);
-        $this->bean->save();
+        global $current_user;
+        global $app_strings;
 
-        $this->bean->handleMultipleFileAttachments();
+        $this->bean = $this->bean->populateBeanFromRequest($this->bean, $_REQUEST);
+        $inboundEmailAccount = new InboundEmail();
+        $inboundEmailAccount->retrieve($_REQUEST['inbound_email_id']);
```

Comment:
There seems to be multiple uses of the inbound_email_id and inbound_email_record parameter. inbound_email_id is not returning email id and is instead returning null.